components/templates/src/global_fns/load_data.rs

```diff
@@ -117,6 +119,10 @@ impl DataSource {
                 .map_err(|e| format!("`load_data`: Failed to parse {} as url: {}", url, e).into());
         }
 
+        if let Some(str) = literal_arg {
```

Comment:
I thought `str` was a keyword, I didn't realise it could be used like that :o

Reply:
Um, haha, I read the rust book a few years ago, so I don't really remember a lot. Hopefully `string_literal` is ok? I'm open to learning the naming convention of variables/constants that are strings.